src/main/java/uk/gov/di/config/Configuration.java

```diff
@@ -109,4 +113,16 @@ private static String configSource() {
     private static String defaultClientId() {
         return System.getenv().getOrDefault("DEFAULT_CLIENT_ID", null);
     }
+
+    public static Stream<Map<String, Serializable>> getClientsPublicKeys() {
+        return getInstance().values().stream()
+                .map(
+                        rpConfig ->
+                                Map.of(
+                                        "client_id",
+                                        rpConfig.clientId(),
+                                        "public_key",
+                                        new PrivateKeyReader(rpConfig.clientPrivateKey())
+                                                .getPublicKey()));
```

Comment:
I reckon we could improve this by putting a method on `RPConfig` to encapsulate this behaviour, something like: 
```
public Map<String, Serializable> jwksConfiguration() {
  return Map.of("client_id", clientId() .... etc);
}
```

so this callsite becomes `getInstance().values().stream().map(RPConfig::jwksConfiguration)`.

Reply:
This looks a lot nicer, have added this method in!